src/app/App.js

```diff
@@ -70,7 +63,7 @@ module.exports = class App {
                 delete process.env.HAKUNEKO_PORTABLE;
             }
             await this._electron.launch();
-            await this._electron.loadHTML(loadingPage);
+            await this._electron.loadURL(this._configuration.applicationCubeURL);
```

Comment:
I suggest to always load `hakuneko://cache/loader.html` instead of making it configurable